src/app/admin/page.tsx

```diff
@@ -39,14 +40,31 @@ const getUsers = async (): Promise<UserRow[]> => {
     });
 };
 
+const getCollectionCentres = async (): Promise<Schema["collection_centres"][]> => {
+    const { data, error } = await supabase.from("collection_centres").select();
+
+    // TODO VFB-23 Move error handling of this request to client side
+    if (error) {
+        throw new Error("Unable to fetch the collection centres information.");
```

Comment:
Can you add `Please try again later` so that it is standard with all the other errors I am adding

Reply:
?

Reply:
Done